stdlib/3/optparse.pyi

```diff
@@ -0,0 +1,248 @@
+    def indent(self) -> None: ...
+    def set_long_opt_delimiter(self, delim) -> None: ...
+    def set_parser(self, parser) -> None: ...
+    def set_short_opt_delimiter(self, delim) -> None: ...
+    def store_option_strings(self, parser) -> None: ...
+
+class IndentedHelpFormatter(HelpFormatter):
+    _long_opt_fmt = ...  # type: str
+    _short_opt_fmt = ...  # type: str
+    current_indent = ...  # type: int
+    default_tag = ...  # type: str
+    help_position = ...  # type: int
+    help_width = ...  # type: Optional[int]
+    indent_increment = ...  # type: Any
+    level = ...  # type: int
+    max_help_position = ...  # type: int
+    option_strings = ...  # type: Dict[Any, Any]
+    parser = ...  # type: Optional[OptionParser]
+    short_first = ...  # type: Any
+    width = ...  # type: Any
+    def __init__(self, *args, **kwargs) -> None: ...
+    def format_heading(self, heading) -> str: ...
+    def format_usage(self, usage) -> str: ...
+
+class Option:
+    ACTIONS = ...  # type: Tuple[str, ...]
+    ALWAYS_TYPED_ACTIONS = ...  # type: Tuple[str, ...]
+    ATTRS = ...  # type: List[str]
+    CHECK_METHODS = ...  # type: Union[None, List[Callable]]
+    CONST_ACTIONS = ...  # type: Tuple[str, ...]
+    STORE_ACTIONS = ...  # type: Tuple[str, ...]
+    TYPED_ACTIONS = ...  # type: Tuple[str, ...]
+    TYPES = ...  # type: Tuple[str, ...]
+    TYPE_CHECKER = ...  # type: Dict[str, Callable]
+    _long_opts = ...  # type: List[str]
+    _short_opts = ...  # type: List[str]
+    action = ...  # type: str
+    dest = ...  # type: Any
+    nargs = ...  # type: int
+    type = ...  # type: Any
+    def __init__(self, *args, **kwargs) -> None: ...
+    def __repr__(self) -> str: ...
+    def __str__(self) -> str: ...
+    def _check_action(self) -> None: ...
+    def _check_callback(self) -> None: ...
+    def _check_choice(self) -> None: ...
+    def _check_const(self) -> None: ...
+    def _check_dest(self) -> None: ...
+    def _check_nargs(self) -> None: ...
+    def _check_opt_strings(self, opts) -> Any: ...
+    def _check_type(self) -> None: ...
+    def _set_attrs(self, attrs: Dict[str, Any]) -> None: ...
+    def _set_opt_strings(self, opts) -> None: ...
+    def check_value(self, opt, value) -> Any: ...
+    def convert_value(self, opt, value) -> Any: ...
+    def get_opt_string(self) -> str: ...
+    def process(self, opt, value, values: Any, parser: OptionParser) -> int: ...
+    def take_action(self, action, dest, opt, value, values, parser: OptionParser) -> int: ...
+    def takes_value(self) -> bool: ...
+
+make_option = Option
+
+class OptionContainer:
+    _long_opt = ...  # type: Dict[str, Any]
+    _short_opt = ...  # type: Dict[str, Any]
+    conflict_handler = ...  # type: Any
+    defaults = ...  # type: Dict[str, Any]
+    description = ...  # type: Any
+    option_class = ...  # type: Any
+    def __init__(self, option_class, conflict_handler, description) -> None: ...
```

Comment:
Amusingly, the first thing this constructor does is call the method`_create_option_list`, which doesn't exist on this class (but it exists on `OptionGroup`, a subclass). It is impossible to instantiate this class.